src/tasks/task-com-equality.cpp

```diff
@@ -38,12 +38,27 @@ namespace tsid
       m_p_com.setZero(3);
       m_v_com.setZero(3);
       m_a_des_vec.setZero(3);
+      m_mask.resize(3);
+      m_mask.fill(1.);
+      setMask(m_mask);      
+    }
+
+
+    void TaskComEquality::setMask(math::ConstRefVector mask)
+    {
+      assert(mask.size() == 3);
+      TaskMotion::setMask(mask);
+      int n = dim();
+      m_constraint.resize(n, m_robot.nv());
+      m_p_error_masked_vec.resize(n);
+      m_v_error_masked_vec.resize(n);
+      m_drift_masked.resize(n);
```

Comment:
`m_drift_masked` is a Vector3, and here you're resizing it to a value that could be smaller than 3. How can that work?

Reply:
Good catch! I don't know. I guess we should change it to Vector.

Reply:
With Eigen, it can works in Release mode, and you don't have any resize.
On the other side, in Debug mode, it should assert ;)

Reply:
But you can keep just remove this line, as soon as you have another way to store the current size mask.

Reply:
I just copy-pasted the SE3 task... I think it's better to stay consistent.